src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -544,6 +545,18 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
                 {0, {ngraph::element::u8, ngraph::element::i8}},
                 {1, {ngraph::element::i8}}
             }),
+            // Common restriction for 2 inputs
+            /*
+            PrecisionsRestriction::create<ngraph::opset5::LSTMSequence>({
+                {{0, 1}, {ngraph::element::u8, ngraph::element::i8}},
```

Comment:
Usage examples should be placed inside PrecisionsRestriction class description (precisions_restriction.hpp)

Reply:
Usage examples are moved to PrecisionRestriction class description